Exec/SCIENCE/ecsn/input_ecsn

```diff
@@ -1,10 +1,10 @@
 # INITIAL MODEL
-maestro.model_file = "kepler_xinlong_ONe6040-final.hse.5120"
-maestro.drdxfac = 5
+maestro.model_file = "ECSN-ONe6040-final.hse.20480"
+maestro.drdxfac = 80
 
```

Comment:
we really don't recommend drdxfac that high.  You should use the normal factor of 5.  There won't be enough information to get a meaningful average when you use 80.

Reply:
I see. I will do 'maestro.drdxfac = 5'.

Reply:
When I use 'maestro.drdxfac = 5', this two below will not be equal. Is this fine?

model file mapping (spherical base state)
 dr of MAESTRO base state =                                 78125.00000
 dr of input file data =                                    9765.625000
@dwillcox @zingale

Reply:
no.  You need to regenerate the initial model at the base state resolution.

Basically, drdxfac = 5 means that the base state resolution will be 5x finer than whatever the finest level's resolution (dx) is.

Reply:
Yeah, I talked to Don and understand.now.